.github/workflows/run-integration-tests.yml

```diff
@@ -0,0 +1,676 @@
+          Set "on_failure" to send a notification only on test failure'
+        required: false
+        type: string
+      slack_notification_after_tests_channel_id:
+        description: "Slack channel ID to send the notification to"
+        required: false
+        type: string
+      slack_notification_after_tests_name:
+        description: "Name of the slack notification"
+        required: false
+        type: string
+      slack_notification_after_tests_notify_user_id_on_failure:
+        description: "Set Slack user id to notify on test failure"
+        required: false
+        type: string
+      test_log_upload_on_failure:
+        description:
+          'Set to "true" to upload the test log on failure as Github artifact'
+        required: false
+        type: boolean
+        default: true
+      test_log_upload_retention_days:
+        description: "Number of days to retain the test log. Default is 3 days"
+        required: false
+        type: number
+        default: 5
+      test_log_level:
+        description: 'Set the log level for the tests. Default is "debug"'
+        required: false
+        type: string
+        default: debug
+      upload_cl_node_coverage_artifact:
+        description:
+          'Set to "true" to upload Chainlink node coverage artifact to as Github
+          artifact'
+        required: false
+        type: boolean
+        default: false
+      upload_cl_node_coverage_artifact_prefix:
+        description: "Prefix for the Chainlink node coverage artifact"
+        required: false
+        type: string
+      SLACK_CHANNEL:
+        description:
+          "SLACK_CHANNEL env used to send Slack notifications from test code"
+        required: false
+        type: string
+      SLACK_USER:
+        description:
+          "SLACK_USER env used to send Slack notifications from test code"
+        required: false
+        type: string
+    outputs:
+      test_results:
+        description: "Test results from all executed tests"
+        value: ${{ jobs.after_tests.outputs.test_results }}
+    secrets:
+      TEST_SECRETS_OVERRIDE_BASE64:
+        required: false
+      QA_AWS_REGION:
+        required: true
+      QA_AWS_ROLE_TO_ASSUME:
+        required: true
+      QA_AWS_ACCOUNT_NUMBER:
+        required: true
+      PROD_AWS_ACCOUNT_NUMBER:
+        required: true
+      GH_TOKEN:
+        required: true
+      AWS_REGION:
+        required: true
+      AWS_OIDC_IAM_ROLE_VALIDATION_PROD_ARN:
+        required: true
+      SLACK_BOT_TOKEN:
+        required: false
+      # Use instead of slack_notification_after_tests_channel_id if channel id is secret
+      SLACK_NOTIFICATION_AFTER_TESTS_CHANNEL_ID:
+        required: false
+      # Used in some tests to send slack notifications
+      SLACK_API_KEY:
+        required: false
+      # Used in some tests to send slack notifications
+      SLACK_CHANNEL:
+        required: false
+
+env:
+  CHAINLINK_IMAGE:
```

Comment:
If these are more traditional integration tests, are we still using Chainlink docker images? I assumed these would be in-memory?